.github/workflows/pull_request_test.yml

```diff
@@ -0,0 +1,15 @@
+name: Automated testing workflow
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the stack
+        run: |
+          cd ./src/cc_catalog_airflow
+          cp env.template .env
+          docker-compose up -d
```

Comment:
Does this image come with `docker-compose`?  I don't think it does.

Reply:
It runs the tests fine I believe.
Following is the output:
![output1](https://user-images.githubusercontent.com/49491061/78019569-949a6f80-736d-11ea-8bf1-b4d27725464f.png)
![output2](https://user-images.githubusercontent.com/49491061/78019577-97956000-736d-11ea-9288-40f3ccdcf43d.png)

Reply:
Even I could not get it running with act but the <a href="https://github.com/creativecommons/cccatalog/pull/344/checks">check</a> executed successfully

Reply:
Yes it comes with docker-compose as per https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md.